### ISSUE TITLE
build failure with latest nightly

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang-nursery/rust-clippy#2550]
building with
````
rustc 1.26.0-nightly (5508b2714 2018-03-18)
````
fails as follows:
````
error[E0308]: mismatched types
  --> clippy_lints/src/unsafe_removed_from_name.rs:53:44
   |
53 |             unsafe_to_safe_check(old_name, new_name, cx, span);
   |                                            ^^^^^^^^ expected struct `syntax::ast::Ident`, found enum `std::option::Option`
   |
   = note: expected type `syntax::ast::Ident`
              found type `std::option::Option<syntax::ast::Ident>`
error: aborting due to previous error
````

**Comment by matthiaskrgr** [comment]
Oh, this is probably already handled in #2549, I didn't see that, sorry for the noise :(

**Comment by oli-obk** [comment]
published 0.0.189
